command_line_assistant/commands/chat.py

```diff
@@ -502,6 +502,14 @@ class InteractiveChatOperation(BaseChatOperation):
 
     def execute(self) -> None:
         """Default method to execute the operation"""
+
+        terminal_file_lock = NamedFileLock(name="terminal")
+
+        if terminal_file_lock.is_locked:
+            raise ChatCommandException(
+                "Interactive chat mode is not available while terminal capture is active."
```

Comment:
We should advise the user further what they can do in this situation. "Type 'exit' to finish capturing terminal input." or "Run 'c -w 1' to ask a question using the previous command." or something along those lines.

Reply:
```suggestion
                "Interactive chat mode is not available while terminal capture is active. To exit the current terminal capture session, press Ctrl + D."
```